include/evm_runtime/evm_contract.hpp

```diff
@@ -91,6 +91,8 @@ class [[eosio::contract]] evm_contract : public contract
    [[eosio::action]] void bridgereg(eosio::name receiver, const eosio::asset& min_fee);
    [[eosio::action]] void bridgeunreg(eosio::name receiver);
 
+   [[eosio::action]] void assertnonce(eosio::name receiver, uint64_t nonce);
```

Comment:
Call the first argument `account` rather than `receiver`.
Call the second argument `next_nonce` rather than `nonce`.

Similar changes in the definition and the test functions that use the action.

Reply:
done